src/uikit.cs

```diff
@@ -37227,6 +38140,11 @@ interface UIViewControllerTransition {
 		[Export ("zoomWithOptions:sourceViewProvider:")]
 		UIViewControllerTransition Zoom ([NullAllowed] UIZoomTransitionOptions options, Func<UIZoomTransitionSourceViewProviderContext, UIView> sourceViewProvider);
 
+		[iOS (26, 0), TV (26, 0), MacCatalyst (26, 0)]
+		[Static]
+		[Export ("zoomWithOptions:sourceBarButtonItemProvider:")]
+		UIViewControllerTransition Zoom ([NullAllowed] UIZoomTransitionOptions options, Func<UIZoomTransitionSourceViewProviderContext, UIBarButtonItem> sourceBarButtonItemProvider);
```

Comment:
Use a separate delegate type, which makes it possible to specify nullability (looks like at least the return value should be nullable).